provider/datagen/src/transform/cldr/datetime/symbols.rs

```diff
@@ -11,7 +11,7 @@ use tinystr::{tinystr, TinyStr16, TinyStr4};
 
 pub fn convert_dates(other: &cldr_serde::ca::Dates, calendar: &str) -> DateSymbolsV1<'static> {
     DateSymbolsV1 {
-        months: other.months.get(&get_month_code_map(calendar)),
+        months: other.months.get(&(get_month_code_map(calendar), calendar)),
```

Comment:
You need to pass in an empty slice here because you can't call `get_month_code_map` with Hebrew

```suggestion
        months: other.months.get(&(&[], calendar)),
```